app/views/calculators/take_cash_in_chunks/_estimate.html.erb

```diff
@@ -1,22 +1,44 @@
-<p>
-  If you take <span class="t-calculator-chunk"><%= number_to_currency(estimate.chunk) %></span> from your
-  <span class="t-calculator-pot"><%= number_to_currency(estimate.pot) %></span> pot, you could get:
+<div id="js-estimate">
+  <p>
+    If you take <span class="t-calculator-chunk"><%= number_to_currency(estimate.chunk) %></span> from your
+    <span class="t-calculator-pot"><%= number_to_currency(estimate.pot) %></span> pot, you could get:
+
+    <strong class="calculator__estimate__number t-calculator-chunk-remaining">
+      <%= number_to_currency(estimate.chunk_remaining) %>
+    </strong>
 
-  <strong class="calculator__estimate__number t-calculator-chunk-remaining">
-    <%= number_to_currency(estimate.chunk_remaining) %>
-  </strong>
+    We estimate you’ll pay:
 
-  We estimate you’ll pay:
+    <strong class="calculator__estimate__number t-calculator-chunk-tax">
+      <%= number_to_currency(estimate.chunk_tax) %> in tax
+    </strong>
 
-  <strong class="calculator__estimate__number t-calculator-chunk-tax">
-    <%= number_to_currency(estimate.chunk_tax) %> in tax
-  </strong>
+    Leaving:
 
-  Leaving:
+    <strong class="calculator__estimate__number t-calculator-pot-remaining">
+      <%= number_to_currency(estimate.pot_remaining) %> in your pot
+    </strong>
+  </p>
+</div>
 
-  <strong class="calculator__estimate__number t-calculator-pot-remaining">
-    <%= number_to_currency(estimate.pot_remaining) %> in your pot
-  </strong>
+<p>
+  <label for="slider">or try a different cash chunk:</label>
+  <%
+  max = estimate.chunk.to_f * Calculators::ARBITRARY_MULTIPLIER
+  if max >= estimate.pot.to_f
+    max = estimate.pot.to_i - 1
+  end
```

Comment:
feel this calculation does not belong in the view, either in the estimate class or maybe a helper, this would also allow the constant to sit next to the calculation.